clustered/integration-test/build.gradle

```diff
@@ -28,7 +28,6 @@ dependencies {
   testCompile project(':clustered:clustered-dist')
   testCompile project(':management')
   testCompile "org.terracotta.management.dist:management-client:$parent.managementVersion"
-  testCompile "org.terracotta.management:monitoring-service-entity:$parent.managementVersion"
```

Comment:
no need anymore: we cab directly use the tms entity to read the monitoring state